lib/main.js

```diff
@@ -126,6 +127,36 @@ ava.test.before(async () => {
     return await resinio.getDeviceStatus(placeholder.uuid) === 'Idle'
   })
 
+  if (options.proxySocks) {
+    console.log('Forwarding SOCKS proxy')
+    await resinio.sshHostOS('mkdir -p /mnt/boot/system-proxy/',
+      context.uuid,
+      context.key.privateKeyPath
+    )
+    console.log('this was that')
+    // await resinio.sshHostOS('cat <<\'EOF\'>> /mnt/boot/system-proxy/redsocks.conf\n' +
```

Comment:
Can should probably create the dir above and put this file in one single command, so we don't open and close an SSH connection twice